astropy/coordinates/sky_coordinate.py

```diff
@@ -688,6 +688,69 @@ def separation_3d(self, other):
         distval = (dx.value ** 2 + dy.value ** 2 + dz.value ** 2) ** 0.5
         return Distance(distval, dx.unit)
 
+    def spherical_offsets_to(self, tocoord):
+        r"""
+        Computes angular offsets to go *from* this coordinate *to* another.
+
+        Parameters
+        ----------
+        tocoord : `~astropy.coordinates.BaseCoordinateFrame`
+            The coordinate to offset to.
+
+        Returns
+        -------
+        lon_offset : `~astropy.coordinates.Angle`
+            The angular offset in the longitude direction (i.e., RA for
+            equatorial coordinates).
+        lat_offset : `~astropy.coordinates.Angle`
+            The angular offset in the latitude direction (i.e., Dec for
+            equatorial coordinates).
+
+        Raises
+        ------
+        ValueError
+            If the ``tocoord`` is not in the same frame as this one. This is
+            different from the behavior of the `separation`/`separation_3d`
+            methods because the offset components depend critically on the
+            specific choice of frame.
+
+        Notes
+        -----
+        By design, this method follows the traditional definition of offsets
+        defined as:
+
+        .. math::
+
+            \Delta {\rm Dec} = \delta_2 - \delta_1
```

Comment:
Use the more general  Lon / Lat here in the docstring instead of RA / DEC?

It's perfectly OK to compute this spherical offset in a GLON / GLAT or ALT / AZ frame, right?